requirements.txt

```diff
@@ -1,2 +1,58 @@
-jira>=3.0.1<3.1.0
-PyGithub>=1.55<1.56
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
```

Comment:
Is this line break intentional?

Reply:
This file is generated automatically by `pip-compile`. The content shouldn't be touched :)